jovo_cv.tex

```diff
@@ -379,6 +379,37 @@ \section{Past Funding}
 
 % \subsection{Past Funding}
 
+
+\cventry{1/19 -- 12/20}{Scalable Cyberinfrastructure to Accelerate Learning the Rules Governing Brain Morphome-Connectome from Genome via Data Integration and Analyses Across Species, Scales and Modalities}{NSF}{Miller (PI), Pending}{}{}
```

Comment:
this section is supposed to be past funding.  you included a bunch of ongoing funding.

Reply:
to be clear, past funding means the end fund date has passed